static_src/constants.js

```diff
@@ -267,9 +269,9 @@ const userActionTypes = keymirror({
   // Action when something goes wrong in user invite and email process.
   USER_INVITE_ERROR: null,
   // Action to display an invite notification
-  USER_INVITE_STATUS_DISPLAYED: null,
+  USER_INVITE_STATUS_CREATED: null,
```

Comment:
This was intentional. I moved away from having multiple constants that result in a notification being displayed. Instead, there are multiple constants that lead to the preparation of a notification, but only one actual constant that leads to the creation of a notification. As a result, this one is changed to reflect the preparation of the invite specific notification.